staging/src/k8s.io/api/admissionregistration/v1/types.go

```diff
@@ -307,6 +307,18 @@ type ValidatingWebhook struct {
 	// include any versions known to the API Server, calls to the webhook will fail
 	// and be subject to the failure policy.
 	AdmissionReviewVersions []string `json:"admissionReviewVersions" protobuf:"bytes,8,rep,name=admissionReviewVersions"`
+
+	// MatchConditions is a list of conditions on the AdmissionRequest ('request') that must be met
+	// for a request to be sent to this webhook. All conditions in the list must evaluate to TRUE for
+	// the request to be matched. This is an alpha feature and managed by the AdmissionWebhookMatchConditions feature gate.
```

Comment:
```
This is an alpha feature and managed by the AdmissionWebhookMatchConditions feature gate.
```

nit: move this to the end of the text, on it's own line (before the `+directive` portion)

Reply:
moved